tee-worker/omni-executor/Makefile

```diff
@@ -81,4 +81,8 @@ stop-local:
 
 .PHONY: get-pallet-metadata
 get-pallet-metadata:
-	subxt metadata --url http://localhost:9944 --allow-insecure --pallets OmniAccount,Teebag > parentchain/artifacts/rococo-omni-account.scale
+	subxt metadata --url http://localhost:9944 --allow-insecure --pallets OmniAccount,Teebag > parentchain/api-interface/artifacts/rococo-omni-account.scale
```

Comment:
We don't have rococo anymore, it's either Litentry or Paseo (depending on how you launch the parachain)

Maybe it's just an intermediate file, but we hope to be accurate when possible : )

Reply:
True, I'll change it to use a generic name 👍🏼